### ISSUE TITLE
"Could not find sass module" error-message

### ISSUE THREAD

**Issue by thebarty** [lookback/meteor-emails#86]
Hi guys,

I am currently using the latest version 0.7.7 on meteor 1.4.1.

Within my packages.json I got ``node-sass`` required like
```
  "dependencies": {
    "node-sass": "^3.13.0",
  },
```

Still I am seeing this error in the logs (NOTE: the email still is send out, it is just a warning-message)
```
Could not find sass module. Sass support is opt-in since lookback:emails@0.5.0.
W20161213-15:50:34.494(1)? (STDERR) 
W20161213-15:50:34.494(1)? (STDERR) Please run `meteor npm install --save node-sass` in your app to add sass support. mailer-utils
W20161213-15:50:34.516(1)? (STDERR) Could not find sass module. Sass support is opt-in since lookback:emails@0.5.0.
W20161213-15:50:34.517(1)? (STDERR) 
W20161213-15:50:34.517(1)? (STDERR) Please run `meteor npm install --save node-sass` in your app to add sass support. mailer-utils
```

The funny thing is, that this error disappears, if I clone the lookback:emails package into my local ``/packages`` directory by doing this in a terminal:
```
cd packages
git clone https://github.com/lookback/meteor-emails.git
```

Bad stuff... I have not checked how this acts on deployment...

Anyone else seeing this?

**Comment by johanbrook** [comment]
Weird, that should've been taken care of [in here](https://github.com/lookback/meteor-emails/blob/master/lib/utils.js#L13-L23).

**Comment by thebarty** [comment]
yeah really really weird... when I simply add something like this on the ``/server/test.js``

```
console.log(require('node-sass'))
```

then I am getting a valid result.

And at the same time it works correctly, if I clone the package to my project folder, so that'll be my workaround for now.

**Comment by zeroasterisk** [comment]
I am also seeing this in my logs too.  I have `node-sass@4.2.0` installed and it does work, but warns me that it's not going to.   wierd.

**Comment by johanbrook** [comment]
I wonder if it's some kind of race condition, but seems unlikely.
